### DISCUSSION TITLE
History Graph zeigt nur die erste Linie an

### DISCUSSION THREAD

**Discussion by jpakusch** [Zefau/ioBroker.jarvis#2649] [Q&A]
Hallo,

ich versuche mich seit kurzem an den History Graphs in Jarvis.
Wenn ich mehrere in einem Diagramm anzeigen will wird aber immer nur die erste angezeigt:
![Screenshot 2024-06-04 104011](https://github.com/Zefau/ioBroker.jarvis/assets/11568426/26180d7f-3553-4d6a-8d8a-ff677f1716c0)
Die Einstellungen sehen so aus:
![Screenshot 2024-06-04 104031](https://github.com/Zefau/ioBroker.jarvis/assets/11568426/de75158a-f86e-46d0-926a-005708b7a633)
Wenn ich die Reihenfolge der Linien hier durch Ziehen ändere wird die andere angezeigt die dann an der ersten Stelle steht.

Ich habe meines Wissens schon alle Tutorials durchgearbeitet, alle issues die ich zu ähnlichen Themen gefunden habe.
Könnt Ihr mir auf die Sprünge helfen?
Danke!

Gruß, Jan

**Comment by mcuiobroker** [comment]
Bitte die Konfigurationen der y-Achse und x-Achse reinstellen.

**Comment by jpakusch** [comment]
Danke für die schnelle Reaktion!
y-Achse:
[{ "min": "dataMin", "max": "dataMax", "maxInterval": 10, "name": "GradC", "nameLocation": "middle", "nameGap": 40 }, { "min": "dataMin", "max": "dataMax", "maxInterval": 10, "name": "Humidity", "nameLocation": "middle", "nameGap": 40 }]

x-Achse:
{ "show": true, "position":"bottom", "splitNumber": 10, "splitLine": { "show": true}, "type": "time"}

**Comment by mcuiobroker** [comment]
Was auffällt, namelocation und namegap sind gleich. Das geht nicht  Eins left und das andere right

**Comment by jpakusch** [comment]
Hallo,
ich habe das mal angepasst.
Das hatte aber keine Verbesserung gebracht.
Was geholfen hatte: ich habe den Graph mit den Solltemperaturen rausgeworfen. jetzt wird Isttemperatur und Feuchte wie erwartet angezeigt.
Keine Ahnung was ihn an den Solltemperaturen gestört hat. LÖeer war der definitiv nicht, ich habe vorhin extra den wert geändert um Werte im Zeitraum zu haben.
Aber ich bin erstmal einen Schritt weiter, Danke dafür!
